### DISCUSSION TITLE
new recommender filament components

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#10250] [Ideas]
- Accordion
- Alert
- Progress
- List Group
- Banner
- Carousel
- Timeline

**Forms:**

- Range
- Floating Label

**Overlays:**

- Popover

**Layout:**

- Skeleton
- Divider

**Comment by jeffegiovani** [comment]
Floating Label 😍

**Comment by atmonshi** [comment]
most of these (if not all) are available as a plugin.
